js/src/components/contact-information/phone-number-card/constants.js

```diff
@@ -0,0 +1,2 @@
+export const VERIFICATION_METHOD_SMS = 'SMS';
+export const VERIFICATION_METHOD_PHONE_CALL = 'PHONE_CALL';
```

Comment:
:nail_care: :nail_care: 
WDYT of changing it to enum? so the person importing the constant, would be sure, they imported everything.
```suggestion
/**
 * …
 *
 * @enum {string}
 */
export const VERIFICATION_METHOD = Object.freeze( {
	SMS: 'SMS',
	PHONE_CALL: 'PHONE_CALL',
} );
```
Also, to emphisize the difference between constant name ('VERIFICATION_METHOD') and values available ('SMS', 'PHONE_CALL')

Reply:
This is nice, I like `VERIFICATION_METHOD.SMS` and `VERIFICATION_METHOD.PHONE_CALL` 🙂  👍

Reply:
Thanks. I did get stuck on their naming for a while before. Your suggestion was one of my candidates for naming and make me more confident to take it. Changed by https://github.com/woocommerce/google-listings-and-ads/pull/993/commits/d59acf956654a6aa293e60469d4b85073d932b16